components/dialog/themes/express.css

```diff
@@ -0,0 +1,3 @@
+.spectrum--express {
+  --spectrum-dialog-confirm-title-text-font-weight: 900;
```

Comment:
I think these theme specific values are not necessary if you use the token `--spectrum-heading-sans-serif-font-weight` instead. It looks like this token is set to `--spectrum-bold-font-weight` for spectrum and `--spectrum-black-font-weight` for express.

Reply:
done!

Reply:
It looks like these still need to be removed, now that they are no longer used (both express.css and spectrum.css).